pkg/exporter/process.go

```diff
@@ -127,6 +133,30 @@ func InitExportingProcess(input ExporterInput) (*ExportingProcess, error) {
 		templateRefCh:   make(chan struct{}),
 	}
 
+	// Start a goroutine for checking whether connection to collector is still open
+	if input.CollectorProtocol == "tcp" {
```

Comment:
is this not an issue with UDP?

Reply:
For udp, we cannot check whether the connection is open by read (probably because of its stateless property), so only set the goroutine for tcp.

Reply:
Will the connection be reset quickly in the case of UDP or we do write on the closed connection too?

Reply:
Had some experiments on Antrea, the connection will be reset quickly so we cannot check the right connection for sending the records.